crates/db_schema/src/source/community_community_follow.rs

```diff
@@ -0,0 +1,11 @@
+use crate::newtypes::CommunityId;
+use lemmy_db_schema_file::schema::community_community_follow;
+
+#[derive(Clone, Debug, PartialEq, Queryable, Selectable)]
+#[diesel(belongs_to(crate::source::community::Community))]
+#[ diesel(table_name = community_community_follow)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
+pub struct CommunityCommunityFollow {
+  pub community_id: CommunityId,
+  pub follower_id: CommunityId,
+}
```

Comment:
I can't think of any other actions, but it might be good to rename this to `community_community_actions`, and have a `followed_at` column. I spose we could do that later, but @dullbananas might want to chime in.

Reply:
At the moment there is no use for actions in this case as no other data would be stored there. If one of these features gets implemented in the future, its easy enough to change the table without any breaking change.